--- lib/python3.10/site-packages/pptx/oxml/shapes/shared.py	2022-09-22 14:36:23.000000000 +0200
+++ patch/lib/site-packages/pptx/oxml/shapes/shared.py	2022-09-22 14:34:12.000000000 +0200
@@ -168,6 +168,14 @@
         return self._nvXxPr.cNvPr.name
 
     @property
+    def shape_alt_text(self):
+        """
+        Alt text of this shape
+        """
+        return self._nvXxPr.cNvPr.descr
+
+
+    @property
     def txBody(self):
         """
         Child ``<p:txBody>`` element, None if not present
@@ -304,6 +312,7 @@
     hlinkHover = ZeroOrOne("a:hlinkHover", successors=_tag_seq[2:])
     id = RequiredAttribute("id", ST_DrawingElementId)
     name = RequiredAttribute("name", XsdString)
+    descr = OptionalAttribute("descr", XsdString)
     del _tag_seq
 
 
